server/renderer.js

```diff
@@ -91,8 +90,12 @@ const replacer = (key = null, value) => {
   return types.replacer(key, cleanedValue);
 };
 
-exports.render = function(requestUrl, context, preloadedState) {
-  const { head, body } = renderApp(requestUrl, context, preloadedState);
+exports.render = function(requestUrl, context, preloadedState, renderApp, webExtractor) {
+  // Bind webExtractor as "this" for collectChunks call.
+  const collectWebChunks = Comp => webExtractor.collectChunks(Comp);
+  collectWebChunks.bind(webExtractor);
```

Comment:
I guess the line above already handles this?

Reply:
Uh, yeah. I think the least error-prone binding is this one:
```
// Bind webExtractor as "this" for collectChunks call.
const collectWebChunks = webExtractor.collectChunks.bind(webExtractor);
```

I'll change it so.